lms/djangoapps/edxnotes/tests.py

```diff
@@ -531,8 +529,7 @@ def test_search_empty_collection(self, mock_get):
         Tests no results.
         """
         mock_get.return_value.content = json.dumps(NOTES_API_EMPTY_RESPONSE).encode('utf-8')
-        six.assertCountEqual(
-            self,
+        self.assertCountEqual(
```

Comment:
We've removed `self.assert` checks. You should now use `assert len(expr1) == len(expr2)` instead of this.